gradle-typescript/src/main/java/com/gradlets/gradle/typescript/TypeScriptBasePlugin.java

```diff
@@ -129,7 +136,19 @@ private static void createTsConfigTaskForSourceSet(
                     createTsConfigTask.setDescription("Creates vscode tsconfig.json for " + sourceSet.getName());
                     createTsConfigTask
                             .getCompileClasspath()
-                            .set(project.getConfigurations().named(sourceSet.getCompileConfigurationName()));
+                            .from(project.getConfigurations()
+                                    .getByName(sourceSet.getCompileConfigurationName())
+                                    .getIncoming()
+                                    .artifactView(v -> v.getAttributes()
+                                            .attribute(
+                                                    LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE,
+                                                    project.getObjects()
+                                                            .named(
+                                                                    LibraryElements.class,
```

Comment:
Its also pretty weird the the basePlugin depends on behaviour introduced in the TypeScriptPlugin

Reply:
The behaviour introduced in the plugin is onlly for project dependencies. BasePlugin doesn't produce consumable artifacts for projects at all